src/mann_whitney.jl

```diff
@@ -199,8 +199,8 @@ where ``\\mathcal{T}`` is the set of the counts of tied values at each tied posi
 
 Implements: [`pvalue`](@ref)
 """
-function ApproximateMannWhitneyUTest(U::Real, ranks::AbstractVector{T},
-    tie_adjustment::Real, nx::Int, ny::Int, median::Float64) where T<:Real
+function ApproximateMannWhitneyUTest(U::Float64, ranks::AbstractVector{Float64},
+    tie_adjustment::Float64, nx::Int, ny::Int, median::Real)
```

Comment:
The changes to the type restrictions could conceivably be breaking for anyone calling this method directly. AFAICT they shouldn't be needed since the inner constructor will `convert` its arguments to the types of the fields, which permits a larger variety of inputs than `Float64`.